docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+services:
+  web:
+    restart: always
+    build:
+      context: ./server
+      dockerfile: Dockerfile
+    volumes:
+      - ./server:/app
+    ports:
+      - "8000:8000"  # Django server
+      - "8765:8765"  # WebSocket server
+    env_file:
+      - .env
+    depends_on:
+      - db
+    networks:
+      - app_network
+
+  db:
+    image: postgres:13
+    restart: always
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: "kundestyrt_db"
+      POSTGRES_USER: "admin"
+      POSTGRES_PASSWORD: "admin"
+    networks:
+      - app_network
+
+  client:
+    restart: always
+    build:
+      context: .
```

Comment:
why is the context for server "./server" but the context for client is "./client"?

Reply:
Was wondering this myself, but didn't get around to investigating. Looked into it now and seems they can both have the context as their respective directories. I think it was different because different people made the initial Docker setup for client and server